<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Detail_view_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Detail View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="sdate">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>3</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>5</month>
     <day>7</day>
    </datetime>
   </property>
   <property name="time">
    <time>
     <hour>3</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseaction">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Modify</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Export</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="auto_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>자동 추출</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manual_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>수동 추출</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="edate">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>3</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>5</month>
     <day>6</day>
    </datetime>
   </property>
   <property name="time">
    <time>
     <hour>3</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
